rpc/tendermint/core/routes.go

```diff
@@ -211,5 +209,38 @@ func (tmRoutes *TendermintRoutes) BlockchainInfo(minHeight,
 	} else {
 		return r, nil
 	}
+}
 
+func (tmRoutes *TendermintRoutes) ListUnconfirmedTxs() (ctypes.ErisDBResult, error) {
+	// Get all Txs for now
+	r, err := tmRoutes.tendermintPipe.ListUnconfirmedTxs(0)
```

Comment:
As discussed, per incorrect comment in tendermint, `-1` is uncapped, all unconfirmed transactions; 0 means none.  But fix comes in next pr